docs/source/multidc/index.rst

```diff
@@ -5,21 +5,14 @@ Deploying multi-datacenter ScyllaDB clusters in Kubernetes
 Prepare a platform for a multi datacenter ScyllaDB cluster deployment:
 
 .. toctree::
-   :hidden:
```

Comment:
let's also lower maxdepth to 1 to align with what it looked like before

Reply:
done